modules/nw-throughput-troubleshoot.adoc

```diff
@@ -1,50 +1,39 @@
 // Module filename: nw-throughput-troubleshoot.adoc
 // Module included in the following assemblies:
-// * networking/configuring-routing.adoc
+// * networking/routes/route-configuration.adoc
 
+:_content-type: CONCEPT
 [id="nw-throughput-troubleshoot_{context}"]
-= Troubleshooting throughput issues
-Sometimes applications deployed through {product-title} can cause
-network throughput issues such as unusually high latency between
-specific services.
+= Throughput issue troubleshooting methods
 
-Use the following methods to analyze performance issues if pod logs do not
-reveal any cause of the problem:
+Sometimes applications deployed through {product-title} can cause network throughput issues such as unusually high latency between specific services.
 
-* Use a packet analyzer, such as ping or link:http://www.tcpdump.org/[tcpdump]
-to analyze traffic between a pod and its node.
+Use the following methods to analyze performance issues if pod logs do not reveal any cause of the problem:
+
+* Use a packet analyzer, such as ping or tcpdump to analyze traffic between a pod and its node.
```

Comment:
```suggestion
* Use a packet analyzer, such as `ping` or `tcpdump` to analyze traffic between a pod and its node.
```
A quick search of the docs shows `tcpdump` as the common usage with literals. Also, since ping could be used as a verb or as a command, I think it helps to differentiate.